reco_utils/recommender/deeprec/io/nextitnet_iterator.py

```diff
@@ -0,0 +1,269 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
could you add the new algo to the readmes:

- https://github.com/microsoft/recommenders/blob/master/notebooks/00_quick_start/README.md
- https://github.com/microsoft/recommenders/blob/master/README.md

Reply:
added nextitnet to the two readmes